k8s/kube-base/deployment-xdmod.yaml

```diff
@@ -104,6 +110,9 @@ spec:
         - name: vol-xdmod-data
           persistentVolumeClaim:
             claimName: pvc-xdmod-data
+        - name: vol-xdmod-openshift-data
+          persistentVolumeClaim:
+            claimName: pvc-xdmod-openshift-data
```

Comment:
I don't think we can have this here as we don't have RWX PVCs yet.  Your job will work if the job is started on the same node, but should fail if it is started on other nodes.  This might be why it was started 3 times.

Reply:
Ah, okay! I've removed this. The three times thing was actually something different though; my initial cron job scheduler was incorrect, and configured to run every minute!

Reply:
That also makes sense.